testing/bdd/features/validation/padding_data_type.feature

```diff
@@ -0,0 +1,70 @@
+@validation @syntax
+Feature: Padding Element and Datatype testing
+  tts:padding is only allowed on certain elements. values are constrained to one, two, three or four non-negative numbers appended by percentage “%”, c” or “px”, delimited by a space.
+
+  ## Assumes that we can pass <tag> to the examples so that we test that the attribute is only applied to the elements that support it.
```

Comment:
Is this assumption valid? I mean, does this feature file generate all the permutations of scenarios for padding on both style and region?